MetaMask/metamask-extension
# Replace `ReadOnlyInput`

## Explanation

Replace the last two uses of the `ReadOnlyInput` with a div. The idea of using an input for read-only data is silly. We can just put it in the DOM directly instead.

## Screenshots/Screencaps

Both displayed private keys are empty accounts from new wallets.

### Before

![before](https://user-images.githubusercontent.com/2459287/161314797-70870be6-84b6-4ae6-a8d8-2ab71f11b106.png)

### After

![after](https://user-images.githubusercontent.com/2459287/161314790-7d0b4fed-fc66-4606-9f0a-bd995319a7d1.png)

## Manual testing steps

- Open the "Account Options" menu (i.e. the three-veritical-dots menu)
- Click "Account Details"
- Click "Export Private Key"
- Ensure that the account address is displayed properly (this is the first replaced element)
- Enter your password
- Ensure the private key is displayed properly (this is the second replaced element)
- Ensure that you can copy the private key by clicking on it
